src/utils/Timer.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+ *      Copyright (C) 2014-2016 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this Program; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "Timer.h"
+
+using namespace LIBRETRO;
+
+uint64_t Timer::microseconds()
+{
+  return std::chrono::duration_cast<std::chrono::microseconds>(m_clock.now().time_since_epoch()).count();
```

Comment:
Can you split this into two lines with a temporary const variable to make this line readable on small screens? The compiler will optimize out the temporary variable.

Reply:
Split into two lines! Thanks.